src/mapInfoContributorManager.cpp

```diff
@@ -91,10 +91,11 @@ MapInfoProperties MapInfoContributorManager::shortInfo(int roomID, int selection
     TRoom* room = mpHost->mpMap->mpRoomDB->getRoom(roomID);
     if (room) {
         QString areaName = mpHost->mpMap->mpRoomDB->getAreaNamesMap().value(areaId);
-        infoText = QStringLiteral("%1 (%3)\n")
-                           .arg(!room->name.isEmpty() && room->name != QString::number(room->getId()) ? QStringLiteral("%1/%2").arg(room->name, QString::number(room->getId()))
-                                                                                                      : QString::number(room->getId()),
-                                areaName);
+        static const QRegularExpression trailingPunctuation("[.,/]+$");
```

Comment:
Please can we have a `QStringLiteral(`...`)` wrapping around the raw string - to avoid any confusion about translation/no-translation needed...

Reply:
Fixed.